Misc/NEWS.d/next/Library/2024-10-19-13-37-37.gh-issue-125710.FyFAAr.rst

```diff
@@ -0,0 +1 @@
+[Enum] fix hashable<->nonhashable comparisons for member values
```

Comment:
For more information about NEWS format, to see https://devguide.python.org/documentation/markup/

:)

Reply:
If there's something wrong with the news entry, please specify it.  I treat them as plain text.